go.mod

```diff
@@ -20,6 +21,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c
+	github.com/imroc/req v0.3.0
```

Comment:
what benefits does this library provide outside the http library?

Reply:
Http requests in less lines of code. Golang is known for extremely verbose http request code (for example, writing to a request body for a POST request requires opening up a Writer, then writing a string to it). The imroc library does this in less lines: 
```go
	res, err := req.Post(url, req.BodyJSON(sarifMetadata), header)
```

We could implement our own httputils. But I settled for the quicker approach of using a library.